packages/xo-web/src/common/xoa-updater.js

```diff
@@ -381,6 +382,17 @@ class XoaUpdater extends EventEmitter {
     }
   }
 
+  async getReleaseChannels() {
+    try {
+      this._releaseChannels = await this._call('getReleaseChannels')
+      return this._releaseChannels
+    } catch (error) {
+      this._releaseChannels = []
+    } finally {
+      this.emit('releaseChannels', this._releaseChannels)
```

Comment:
I don't see the point of storing them in Redux if they are not refreshed automatically.

Why not simply use the returned value?

Reply:
If we load `releaseChannels` only in `update page`, we will loose'em when we navigate to other pages and comeback later. That's why I found it interesting to have them in store.